### PR TITLE
feat(dedicated): billing module lazy loading

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6316
| License          | BSD 3-Clause

## Description

Lazy load billing section

cc @marie-j

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/activation/activation.module.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
+import angular from 'angular';
+import 'angular-translate';
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import '@ovh-ux/ng-translate-async-loader';
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/activation/activation.module.js:4]: Why sometimes `import foo from 'bar'` and sometimes `import 'bar'` ? (apart from when there is no choice)

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/cancel-resiliation/translations/Messages_es_ES.json [hunk 2]
```diff
@@ -1,3 +0,0 @@
-{
-  "autorenew_service_cancel_resiliation_success": "Se ha cancelado la baja del servicio: {{ service }}."
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/cancel-resiliation/translations/Messages_es_ES.json:2]: Not used ?

Comment (frenautvh) [reply packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/cancel-resiliation/translations/Messages_es_ES.json:2]: i believe it's not, it's used here : https://github.com/ovh/manager/tree/c645824395828ca08b2352dace31dce34926344a/packages/manager/modules/billing/src/components/cancel-resiliation
(you can see the translation exists on the module)

File: packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/activation/activation.module.js (hunk 1 above)
Comment (frenautvh) [reply packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/activation/activation.module.js:4]: copy paste :/
i know that some older modules doesn't export moduleName but in that case we should be able to ```import foo from bar;``` i'm going to fix it

**Comment by frenautvh** [comment]
closed in favor of : https://github.com/ovh/manager/pull/4433

**Pull request closed without merging**